client/extensions/woocommerce/app/promotions/promotion-create.js

```diff
@@ -11,13 +11,17 @@ import { localize } from 'i18n-calypso';
  * Internal dependencies
  */
 import Main from 'components/main';
+import { fetchPromotions } from 'woocommerce/state/sites/promotions/actions';
```

Comment:
These imports are out of order.

Reply:
I'd like to delay any import rearranging until we've all agreed upon the standard.